eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -309,10 +309,12 @@ object Router {
                         searchMaxFeePct: Double,
                         searchMaxRouteLength: Int,
                         searchMaxCltv: CltvExpiryDelta,
-                        searchHeuristicsEnabled: Boolean,
+                        searchRatioBias: Double,
                         searchRatioCltv: Double,
                         searchRatioChannelAge: Double,
                         searchRatioChannelCapacity: Double,
+                        searchHopCostBase: MilliSatoshi,
+                        searchHopCostMillionths: Long,
```

Comment:
Should we validate that these two are positive values?

Interestingly, if we use negative values here we favor longer routes, which some privacy-aware users may want, so not sure whether it would be a bug or a feature...if we don't constrain it to positive values we probably need tests to verify the behavior with negative values.